<?xml version="1.0"?>
<launch>

    <arg name="fontPath" value="$(find whycon_ros)/etc/font.ttf"/>
    <arg name="calibResPath" value="$(find whycon_ros)/etc/Calib_Results.m"/>
    <arg name="calibDefPath" value="$(find whycon_ros)/etc/default.cal"/>
    
    <node pkg="whycon_ros" type="whycon_ros" name="whycon" output="screen" args="$(arg fontPath) $(arg calibResPath) $(arg calibDefPath)">
        
        <remap from="/cv_camera/image_raw" to="/cv_camera/image_raw" />
<!--        <remap from="/camera/camera_info" to="/cv_camera/camera_info" /> -->
	
	<param name="useGui" value="false"/>
	<param name="saveLog" value="false"/>
	<param name="saveVideo" value="false"/>
	
	<param name="idBits" value="5"/>
        
    </node>

</launch>
